user/languages/smalltalk.md

```diff
@@ -25,32 +24,30 @@ and cc [@bahnfahren](https://github.com/bahnfahren),
 
 ## Basic configuration
 
-For a minimum configuration you need to specify two parameters, the language
-and the baseline of your project.
+For a minimum configuration, you need to specify two parameters, the language and the baseline of your project. Default build configuration is a Squeak-5.0 image on Linux.
 
 ```yaml
-    language: smalltalk
-    env:
-      - BASELINE=MyProjectBaseline
+language: smalltalk
+baseline: MyProjectBaseline
 ```
 
 ## Configuration options
 
-By default Travis CI will use `Squeak5.0` release. It is also possible
-to test projects against `SqueakTrunk`, `Squeak4.6` and `Squeak4.5`. To do so, set the
-`smalltalk` key in `.travis.yml`. For example, to test against both `Squeak5.0` and
-the `SqueakTrunk`:
+Next to the default image it is also possible to test projects against other versions of Squeak/Smalltalk or Pharo (an exhaustive list of supported images can be found at [smalltalkCI's GitHub repository](https://github.com/hpi-swa/smalltalkCI#images)). Also you can test multiple baselines at once.
```

Comment:
Could you reword to something like

"As well as using the default image you can also test test projects against other versions of Squeak/Smalltalk or Pharo (an full list of supported images can be found at [smalltalkCI's GitHub repository](https://github.com/hpi-swa/smalltalkCI#images))."